extkeys/mnemonic.go

```diff
@@ -26,7 +26,7 @@ import (
 // https://github.com/bitpay/bitcore-mnemonic/blob/master/lib/mnemonic.js (used in eth-lightwallet.js)
 
 // Salt is input salt to mnemonic key
-const Salt = "status-im"
+const Salt = "mnemonic"
```

Comment:
As you're not using this outside this package anymore, I think there is no reason to have it exported.